CHANGELOG.md

```diff
@@ -55,6 +55,11 @@ librdkafka.
    Users on Brokers 0.9.x or older should already be configuring
    `api.version.request=false` and `broker.version.fallback=...` so there
    should be no functional change.
+ * The default producer batch accumulation time, `linger.ms`, has been changed
+   from 0.5ms to 5ms to improve batch sizes and throughput while reducing
+   the per-message protocol overhead.
+   Applications that require lower produce latency than 5ms will need to
```

Comment:
pedantically, this is of course more complicated, at higher throughputs they'll get better latency with 5ms linger.ms than 0.5ms. but doesn't matter...